### ISSUE TITLE
How will the migrated playbook information fit with the sytems folders so we avoid confusion and duplication?

### ISSUE THREAD

**Issue by craigmmills** [Vizzuality/playbook#18]
I'm seeing a bunch of sort of guidance in the migrated playbook which might need to be consented

**Comment by agnessa** [comment]
I think it should be better organised to clearly show which content is "new" and which is "legacy". For example they could be broken down into:
- broad systems guidelines
- pre-June 2018 (or legacy) guidelines
- post-June 2018 (or modern) guidelines

Nevertheless, since those older guidelines were prepared before we had the consent process in place, I think we can avoid having to debate them all over and assume they were agreed upon using an earlier version of the approval system: they were hardly ever prepared by one person, and most people who worked on VizzEthos would have read them and had a chance to comment. They can be contested like any other guideline going forward. Also, where someone prepares a new related guideline (e.g. #17), they should check within the legacy content first and migrate over any relevant bits into the "modern" realm (with a system of either removing stuff from legacy or marking as migrated) - this way we might be able to get rid of legacy in the long run and actually consent the migrated bits.

Since this issue is mostly about the status of the VizzEthos content at the moment, I assigned people from the VizzEthos slack channel here. I believe we need to update the meta guideline on these points (content organisation, gradual legacy migration).

**Comment by Jamie-Gibson** [comment]
Yeah makes sense, but don't forget that a lot of the Vizz Ethos guidelines were just never released to the team; for some of us it's legacy, but for the other 29 people in the company it's all completely new. So labelling them as 'legacy' does the danger of them being ignored... 

I think we should leave the Vizz Ethos guidelines where they are for now as provisional guidance, so groups can adopt pieces of them as they see fit (especially on information flow). Once all the systems are finished we can get the rest adopted as functional area guidelines (ie they're a layer below and governed by the system guidelines).

**Comment by agnessa** [comment]
"provisional" is a word which indeed sounds more encouraging then legacy.

**Comment by d4weed** [comment]
I'm not sure we should be focusing on pre and post-June 2018 in order to categorize guidelines. IMO our guideline policy is flexible enough that any guideline can be amended at any point if someone deems it necessary, plus it should be easy to see by looking at a specific file's history if it was just copy-pasted or it went through a process.

I also think it's important to note that there were a bunch of guidelines already in place even before the Vizz-Ethos was written, and the fact that they weren't created through consent shouldn't invalidate them. In fact, I think we should put them all in one place and let people judge when it makes sense to change them. They may not be provisional after all if they work

Some examples of these are

https://sites.google.com/a/vizzuality.com/intranet/resources/expenses
https://sites.google.com/a/vizzuality.com/intranet/resources/travel
https://sites.google.com/a/vizzuality.com/intranet/onboarding
https://sites.google.com/a/vizzuality.com/intranet/guidelines/using-pivotal

We should probably give everyone a heads up these are being added, and allow some time for refining, but other than that I don't see why they should be different as in most cases, as @agnessa notes, they were created by different people in collaboration

**Comment by agnessa** [comment]
Right there on your list is the third guideline I know of that has to do with how to use Pivotal 😱

**Comment by Jamie-Gibson** [comment]
Ok I may do some migration work later this afternoon and announce it tomorrow (in addition to the 2 system guidelines we'll be publishing tomorrow)

**Comment by d4weed** [comment]
hahaha! actually that's probably a copy-paste from pivotal's tutorial with 2 or 3 modifications, we could certainly live without all that text and have a link plus whatever specifics we think relevant
